remote-release-notes/v1_92.md

```diff
@@ -5,3 +5,27 @@
 ### GPG keyboxd support
 
 The GPG forwarding for Dev Containers now supports when GPG's new `keyboxd` service for public keys is used by the local machine, the container or both.
+
+## SSH
+
+### Local Port Range Configuration
+
+The  usable range of local ports is now configurable with the <a href="vscode://settings/remote.SSH.preferredLocalPortRange">`remote.SSH.preferredLocalPortRange`</a> setting.  This may be useful in cases where a range of local ports is reserved on a machine but not yet bound.
```

Comment:
From https://github.com/microsoft/vscode/issues/223027 it looks like we should also add the `codesetting` attribute